doc/classes/BaseMaterial3D.xml

```diff
@@ -120,6 +120,15 @@
 		<member name="backlight_texture" type="Texture2D" setter="set_texture" getter="get_texture">
 			Texture used to control the backlight effect per-pixel. Added to [member backlight].
 		</member>
+		<member name="bent_normal_enabled" type="bool" setter="set_feature" getter="get_feature" default="false">
+			If [code]true[/code], bent normal mapping is enabled. This allows for more accurate specular occlusion and indirect lighting.
+		</member>
+		<member name="bent_normal_texture" type="Texture2D" setter="set_texture" getter="get_texture">
+			Texture used to specify the bent normal at a given pixel. The [member bent_normal_texture] only uses the red and green channels; the blue and alpha channels are ignored. The normal read from [member bent_normal_texture] is oriented around the surface normal provided by the [Mesh].
+			[b]Note:[/b] The mesh must have both normals and tangents defined in its vertex data. Otherwise, the normal map won't render correctly and will only appear to darken the whole surface. If creating geometry with [SurfaceTool], you can use [method SurfaceTool.generate_normals] and [method SurfaceTool.generate_tangents] to automatically generate normals and tangents respectively.
+			[b]Note:[/b] Godot expects the normal map to use X+, Y+, and Z+ coordinates. See [url=http://wiki.polycount.com/wiki/Normal_Map_Technical_Details#Common_Swizzle_Coordinates]this page[/url] for a comparison of normal map coordinates expected by popular engines.
+			[b]Note:[/b] The normals must be cosine weighted in order to function correctly.
```

Comment:
"in order to" is redundant:

```suggestion
			[b]Note:[/b] The normals must be cosine weighted to function correctly.
```

Additionally, it may be useful to point to a resource that tells you how to generate bent normal maps. I assume these can't be generated from existing normal map textures though.

Reply:
Yeah they are separately authored.
I will see if I can find a good resource.

Reply:
I'm pushing the suggested changes for now as I can't find particularly good resources on bent normal generation.
For reference though:
Programs I know can bake them:

- Substance Painter
- Substance Designer
- xNormal (Unsure how exactly this program works)
- Blender (with the [right setup](https://blender.stackexchange.com/questions/157079/how-do-i-bake-a-bent-normal-map-in-blender))